CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Core Text backend has always returned regular style fallback fonts
```

Comment:
```suggestion
- CoreText backend ignoring style for font fallback
```

I think this should better describe what actually got fixed and is desired.

Reply:
OK. But I think that it is not `CoreText` but `Core Text`.

https://developer.apple.com/documentation/coretext?language=objc

Reply:
Good point, let's go with `Core Text` then.

Reply:
Good point, let's go with `Core Text` then.